1224-user_administration/1424-administrator-configures-automatic-creator-role.feature

```diff
@@ -0,0 +1,13 @@
+@1424
+Scenario: Administrator configures automatic Creator role
+    Given that the User is logged in as Administrator
+    And they are on the User Administration Page
+    And they see a list of <Options> in the Registrator section
```

Comment:
Hva er options? Ser det er brukt som eksempler her, men neste setning `And they see that none are selected` gir ikke mening uten å spesifisere hva options er. Er det en checkbox? radio? 
```suggestion
    And they see a list of checkboxes in the Registrator section
    And they see each checkbox has corresponding <Option> text
```